packages/grid/src/elements/Col.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { HTMLAttributes } from 'react';
+import PropTypes from 'prop-types';
+import { StyledCol } from '../styled';
+
+export interface IColProps extends HTMLAttributes<HTMLDivElement> {
+  /** Sizing for all breakpoints. */
+  size?: number | string;
+  /** Sizing for extra small breakpoints. */
+  xs?: number | string | boolean;
+  /** Sizing for small breakpoints. */
+  sm?: number | string | boolean;
+  /** Sizing for medium breakpoints. */
+  md?: number | string | boolean;
+  /** Sizing for large breakpoints. */
+  lg?: number | string | boolean;
+  /** Sizing for extra large breakpoints. */
+  xl?: number | string | boolean;
+  offsetXs?: number | string | boolean;
+  offsetSm?: number | string | boolean;
+  offsetMd?: number | string | boolean;
+  offsetLg?: number | string | boolean;
+  offsetXl?: number | string | boolean;
+  /** Use flexbox alignment utilities to horizontally align */
+  alignSelf?: 'start' | 'center' | 'end';
+  justifyContent?: 'start' | 'center' | 'end' | 'around' | 'between';
+  order?: any;
+}
+
+/**
+ * Accepts all `<div>` attributes and events
+ */
+export const Col = React.forwardRef<HTMLDivElement, IColProps>((props, ref) => (
```

Comment:
I assume we are doing both named and default exports to help allow `styleguidist` to understand the `forwardRef` syntax?

Could we _only_ use the named export when possible? This would align our `elements` usage with our `styled` usages.

Reply:
Oops, this wasn't intentional.

It appears that I can use named exports all the way through and styleguidist still picks them up. Are we sure now is the right time to start pivoting to using a named export pattern? The packages will be in a 50%/50% state.

Reply:
Since we are fully on styleguidist for at least the short-term I think making the move is ok to allow us to keep documenting as we have in the past.

> packages will be in 50%/50% state

With us being in TypeScript land I don't think this hurts us as much since the code-completion and build will help make sure we are using the exports correctly.

Reply:
Nvmd. I lied. The only way the current syntax works with styleguidist is with named exports. Will update...

![Screen Shot 2019-11-18 at 10 02 13 AM](https://user-images.githubusercontent.com/143773/69077453-85340e80-09ea-11ea-921e-10e4edc142c5.png)